src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/BindRequiredAttribute.cs

```diff
@@ -5,9 +5,15 @@
 
 namespace Microsoft.AspNetCore.Mvc.ModelBinding
 {
+    /// <summary>
+    /// Indicates that the property is required for model binding.
```

Comment:
@rynowak FYI your comment [here](https://github.com/aspnet/Mvc/pull/4841/files/c800712b84a3a4ae2f197bf485667262e119bb1d#r66532249) also applies for this.